docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -32,7 +32,7 @@ When a FILTER clause is attached to an aggregate function, only the matching row
 GROUP BY group_expression [ , group_expression [ , ... ] ]
     [ { WITH ROLLUP | WITH CUBE | GROUPING SETS (grouping_set [ , ...]) } ]
 
-GROUP BY GROUPING SETS (grouping_set [ , ...])
+GROUP BY { group_expression | { ROLLUP | CUBE | GROUPING SETS } (grouping_set [ , ...]) } [ , ... ]
```

Comment:
Shall we remove the `[ , ... ]` at the end since we don't support it right now?

Reply:
> Shall we remove the `[ , ... ]` at the end since we don't support it right now?

Syntactically speaking, we can support this after this pr, but currently does not support executio.